app/src/pages/inside/dashboardItemPage/modals/widgetWizardModal/widgetWizardContent/wizardControlsSection/wizardSecondStepForm/wizardSecondStepForm.jsx

```diff
@@ -67,6 +73,7 @@ export class WizardSecondStepForm extends Component {
     return (
       <form onSubmit={handleSubmit(onSubmit)}>
         <ControlsForm
+          onClear={this.clearField}
```

Comment:
I think you can just put `change` method here with `changeFieldValue` property name.
It will be more common and flexible.